packages/libs/react-ui/src/styles/vars.css.ts

```diff
@@ -150,23 +145,29 @@ export const vars = createGlobalTheme(':root', {
     $positiveHighContrast: colorPalette.$green100,
     $positiveAccent: colorPalette.$green60,
 
+    // State
+
     $disabledContrast: hexToRgba(colorPalette.$gray40, 0.6),
+
+    // Legacy (Do not use)
+
+    $background: colorPalette.$gray10,
+    $foreground: colorPalette.$gray100,
+
+    $neutral1: colorPalette.$gray10,
+    $neutral2: colorPalette.$gray20,
+    $neutral3: colorPalette.$gray40,
+    $neutral4: colorPalette.$gray60,
+    $neutral5: colorPalette.$gray90,
+    $neutral6: colorPalette.$gray100,
```

Comment:
Do we want to deprecate these? These are still dynamic depending on the theme, but I'm not sure how well they have been working in context

Reply:
What are your thoughts @sstraatemans and @isaozler

Reply:
At the moment I still use these.
If we have them in another place / name and they are still dynamic then let me know. I can change my components.
But sometimes they are needed.

Reply:
I suggest moving to the new tokens in the ideal world, but I don't know the best development-wise. Because otherwise, it could also create a lot of confusion/chaos...

Reply:
The value in those neutral color tokens is that they change colors depending on the applied theme. If we use `gray10` we will have to manually add a dark theme version. My opinion is that we still want to keep those tokens and that functionality and instead of deprecating them, we just make sure they are using the correct colors

Reply:
I haven't really been able to apply the neutral colours yet with the new colour system though. Because even though almost all of the colours from neutral are in the new colour palette, the light and dark pairs don't match with most of the designs (gray10 light + gray 90 dark instead of gray20 light + gray 90 dark for example)

Reply:
yeah, since @isaozler is out right now, we will have to wait to refine them. I think this is fine and we can revisit this when he gets back

Reply:
Created a task for it on the board: https://app.asana.com/0/1204958723576385/1205169835342679/f